posthog/hogql_queries/insights/stickiness_query_runner.py

```diff
@@ -376,7 +380,9 @@ def setup_series(self) -> list[SeriesWithExtras]:
 
     def date_range(self, series: SeriesWithExtras):
         if series.is_previous_period_series:
-            return self.query_previous_date_range
+            if not series.compare_to:
+                return self.query_previous_date_range
```

Comment:
I would say this `if` would be better encapsulated _within_ `query_compare_to_date_range()`

Reply:
Yes, much cleaner.
![image](https://github.com/PostHog/posthog/assets/1855120/7495eda4-b47e-4897-9b04-026640d65736)